main.js

```diff
@@ -0,0 +1,83 @@
+            document.getElementById("cell-4").addEventListener('click', (event) => { if (event.target.className === "taken") { badTurn.removeAttribute("hidden"); } });
+            document.getElementById("cell-5").addEventListener('click', (event) => { if (event.target.className === "taken") { badTurn.removeAttribute("hidden"); } });
+            document.getElementById("cell-6").addEventListener('click', (event) => { if (event.target.className === "taken") { badTurn.removeAttribute("hidden"); } });
+            document.getElementById("cell-7").addEventListener('click', (event) => { if (event.target.className === "taken") { badTurn.removeAttribute("hidden"); } });
+            document.getElementById("cell-8").addEventListener('click', (event) => { if (event.target.className === "taken") { badTurn.removeAttribute("hidden"); } });
+
+            // Place click Event Listeners for playing turns:
+            document.getElementById("cell-0").addEventListener('click', (event) => { badTurn.setAttribute("hidden", ""); event.target.textContent = whoNow; event.target.className = "taken"; changeTurn(); }, {once: true});
+            document.getElementById("cell-1").addEventListener('click', (event) => { badTurn.setAttribute("hidden", ""); event.target.textContent = whoNow; event.target.className = "taken"; changeTurn(); }, {once: true});
+            document.getElementById("cell-2").addEventListener('click', (event) => { badTurn.setAttribute("hidden", ""); event.target.textContent = whoNow; event.target.className = "taken"; changeTurn(); }, {once: true});
+            document.getElementById("cell-3").addEventListener('click', (event) => { badTurn.setAttribute("hidden", ""); event.target.textContent = whoNow; event.target.className = "taken"; changeTurn(); }, {once: true});
+            document.getElementById("cell-4").addEventListener('click', (event) => { badTurn.setAttribute("hidden", ""); event.target.textContent = whoNow; event.target.className = "taken"; changeTurn(); }, {once: true});
+            document.getElementById("cell-5").addEventListener('click', (event) => { badTurn.setAttribute("hidden", ""); event.target.textContent = whoNow; event.target.className = "taken"; changeTurn(); }, {once: true});
+            document.getElementById("cell-6").addEventListener('click', (event) => { badTurn.setAttribute("hidden", ""); event.target.textContent = whoNow; event.target.className = "taken"; changeTurn(); }, {once: true});
+            document.getElementById("cell-7").addEventListener('click', (event) => { badTurn.setAttribute("hidden", ""); event.target.textContent = whoNow; event.target.className = "taken"; changeTurn(); }, {once: true});
+            document.getElementById("cell-8").addEventListener('click', (event) => { badTurn.setAttribute("hidden", ""); event.target.textContent = whoNow; event.target.className = "taken"; changeTurn(); }, {once: true});
```

Comment:
Your `badTurn` element starts hidden, so you shouldn't need to set it to hidden here. It's not necessarily a bad idea to make sure it's hidden, but it would probably be better to do that in your `changeTurn` function to remove duplication.